source/clear-linux/get-started/bare-metal-install/bare-metal-install.rst

```diff
@@ -1,286 +1,336 @@
+additional tasks during |CL| Installer setup:
+
+* modify the disk layout using the cgdisk utility.
+* add additional command-line parameters to the kernel.
+* create a hostname for your system.
+* create an administrative user.
+* add additional software bundles to enhance the functionality of your initial
+  |CL| installation.
+* optionally, set up a static IP address for your system.
+
+To implement any of the additional items listed above for the
+manual option, select the :guilabel:`< Manual(Advanced) >` menu item and
+follow the steps outlined in the :ref:`bare-metal-manual-install`. This guide
+is a continuation from this point.
+
+.. toctree::
+   :maxdepth: 1
+
+   bare-metal-manual-install
+
+Automatic installation
+**********************
+
+If you only want to install the minimum components for your |CL|
+implementation the recommended action is to select the
+:guilabel:`< Automatic >` menu item shown in figure 7.
+
+#. Move your cursor to this field and press the :kbd:`Enter` key.
+
+#. This will bring you to the ``Choose target device for installation``
+   screen as shown in figure 8. Move your cursor to the desired target and
+   press the :kbd:`Enter` key.
+
+   .. figure:: figures/cai5of6.png
+      :scale: 50 %
+      :alt: Choose target device for installation
+
+      Figure 8: :guilabel:`Choose target device for installation`
+
+   In this example we select the single primary partition from our hard drive.
+
+#. With all the |CL| installer setup information gathered for the automatic
+   installation option, the |CL| Installer will prompt you to begin the actual
+   installation as shown in figure 9.
+
+   .. figure:: figures/cai6of6.png
+      :scale: 50 %
+      :alt: Begin installation
+
+      Figure 9: :guilabel:`Begin installation`
+
+   If you are satisfied with the information you have entered, select the
+   :guilabel:`< Yes >` field and press :kbd:`Enter` to begin installing |CL|.
+
+|CL| Installation begins...each step will show it's status as it progresses
+through the automated installation process.
+
+Complete installation process
+=============================
+
+#. Once all steps have completed, you will see the :guilabel:`< Ok >` prompt
+   as shown in figure 10. Press the
+   :kbd:`Enter` key to continue.
+
+   .. figure:: figures/caisuccess.png
+      :scale: 50 %
+      :alt: Installation complete
+
+      Figure 10: :guilabel:`Installation complete`
+
+#. The final screen is shown in figure 11 and you will be prompted that the
+   installation was successful and the system will be rebooted. Press the
+   :kbd:`Enter` key and remove the USB media while the system restarts.
+
+   .. figure:: figures/cairestart.png
+      :scale: 50 %
+      :alt: Successful installation
+
+      Figure 11: :guilabel:`Successful Installation`
+
+Set up your root and user accounts
+==================================
+
+With the USB device removed and the system restarted, your newly installed

```

Comment:
Those things are not the important prerequisite to setup the accounts. The important prerequisite is the completed installation, be that manual or automatic. Avoid direct references to past sections since you don't know which path the reader followed to get here. I would rewrite this paragraph thus: "Once the |CL| installation is complete and the system boots,  a full screen console requests your ``login:`` as shown in figure 12: